src/StudyViewer/StudyDetails.jsx

```diff
@@ -264,20 +272,24 @@ class StudyDetails extends React.Component {
        <div className='study-details'>
          <Space className='study-viewer__space' direction='vertical'>
            <Space>
-             {this.props.isSingleItemView
-               ? (
-                 <Button
-                   label={'Learn More'}
-                   buttonType='primary'
-                   onClick={() => this.props.history.push(`${this.props.location.pathname}/${encodeURIComponent(this.props.data.rowAccessorValue)}`)}
-                 />
-               )
-               : null}
-             {
-               (this.props.studyViewerConfig.buttons) ? this.props.studyViewerConfig.buttons.map(
-                 (buttonConfig, i) => this.getButton(i, buttonConfig, userHasLoggedIn),
-               ) : null
-             }
+             <div>
```

Comment:
is this change only a linting change?

Reply:
no i added it to eliminate a react duplicate key warning that was flooding the console, i tested the change and it had no visual impact of the page

Reply:
had to remove the divs as it was failing automated testing, replaced with react fragments and hope that works